src/main/java/core/basesyntax/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.database.DataBase;
+import core.basesyntax.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final int INDEX_OF_OPERATION_TYPE = 0;
+    private static final int INDEX_OF_FRUIT_TYPE = 1;
+    private static final int INDEX_OF_FRUIT_QUANTITY = 2;
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public List<String> readDataFromFile(String path) {
+        List<String> fruitData = DataBase.listDb;

```

Comment:
We are reading the file here. Nothing else.
```suggestion
        List<String> fruitData = new ArrayList<>();
```